tests/rptest/tests/full_disk_test.py

```diff
@@ -360,7 +360,7 @@ def test_target_min_capacity_wanted_time_based(self):
         # there is definitely going to be some fuzz factor needed here and may
         # need updated, but after many runs 50mb was a good amount of slack.
         assert diff <= (
-            100 * 2**20
+            115 * 2**20
```

Comment:
I'm okay with bumping this value, but I'm wondering if there are alternative conditions that equally accomplish the similar coverage while being more forgiving. Like, would it defeat the purpose of the test to wrap the storage check in wait_until and use a lower fuzz factor?